### ISSUE TITLE
1 v all winning probabilities

### ISSUE THREAD

**Issue by samyip123** [sublee/trueskill#32]
I'd like to calculate the winning probabilities of a 1 vs all game . Is there a function to derive The probabilities which sums to 1?

**Comment by sublee** [comment]
Do you mean free-for-all games? Or just 2 teams but there's only 1 player on the first team?

**Comment by samyip123** [comment]
For exmaple there are 13 players in the game, they all compete with each other, anyone can win individually, without a draw, thanks !!

**Comment by sublee** [comment]
The original TrueSkill provides only a draw probability function for any structure of matches. Anyways, many programmers who use this module have discussed the win probability function at #1 for a few years. In this issue, I've seen only the function for just 2 teams, not the free-for-all. Currently, I have no idea how to calculate each win probability in a free-for-all match.

**Comment by samyip123** [comment]
Thanks for the help, however is there a fuunction for updating ratings for free for all match? Thanks

**Comment by sublee** [comment]
Yes, there's `rate()` for any kind of matches. 1v1, 2v3, 3v3v7, or 1v1v1v1v1 (free-for-all)
